tide/core/PixelStreamContent.h

```diff
@@ -66,6 +70,12 @@ class PixelStreamContent : public Content
     /** @return true if the streamer can handle aspect ratio changes. */
     bool hasFixedAspectRatio() const override;
 
+    /** Get the current page number. */
+    int getPage() const;
+
+    /** Get the total number of pages. */
```

Comment:
It is only used by the webbrowser and I think it makes sense in this case. I didn't find a more generic name for this. Maybe subclassing PixelStreamContent with a new WebbrowserContent and adding the two functions there would be better?